### PR TITLE
Emotion 11 upgrade

### LINKED ISSUE #8348: Upgrade @emotion/core and @emotion/styled to new major version 11 in Simorgh
**Is your feature request related to a problem? Please describe.**
Upgrade `@emotion/core` and `@emotion/styled` to new major version 11.

Psammead PR 1: https://github.com/bbc/psammead/pull/4193
Psammead PR 2 (psammead-styles): https://github.com/bbc/psammead/pull/4231

Details of any breaking changes can be found here https://github.com/emotion-js/emotion/releases/tag/%40emotion%2Fstyled%4011.0.0

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves https://github.com/bbc/simorgh/issues/8348

**Overall change:** 

- Update all Psammead packages to use Emotion 11. The core migration involved the following changes within root psammead, and then in each package that had one of these dependencies:
```json
    "@emotion/core": "^10.0.35"  >  "@emotion/react": "^11.1.4"
    "@emotion/styled": "^10.0.27"  >  "@emotion/styled": "^11.0.0"
    "jest-emotion": "^10.0.32"  >  "@emotion/jest": "^11.1.0"
    "babel-plugin-emotion": "^10.0.33"  >  "@emotion/babel-plugin": "^11.1.2"
```

- These package updates triggered classNames to be regenerated for snapshots across all packages, and some rearranging of styles. The line count of all snapshots before/after migration is consistent though, with two exceptions:

Two packages had line changes in their snapshots - `psammead-radio-schedule` & `psammead-media-player/Placeholder`.  
- `psammead-media-player/Placeholder` had some janky styles, which were [fixed in this PR](https://github.com/bbc/psammead/pull/4193/files#diff-4c3929e4357421334316b48a45933497fa55ace8cd5e3d16d2a12a6b4d06ae3fR20) and restored snapshot parity
- `psammead-radio-schedule` has lost some snapshot lines but as detailed in Ryan's comments below, we think these are optimisations rather than the loss of styles.

**Notes**
- We removed `psammead-styles` from this singular upgrade PR due to widespread changes in its snapshots.
- `@emotion/core": "^10.1.1"` is also kept as a dependency while we still need to support Emotion 10 for psammead-styles

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -10,6 +10,16 @@ exports[`Media Player: Placeholder should render a video placeholder 1`] = `
   cursor: pointer;
 }
 
+.emotion-1:hover .css-9vh530-StyledPlayButton,
+.emotion-1:focus .css-9vh530-StyledPlayButton {
+  background-color: #B80000;
+}
+
```
Comment (ryanmccombe) [inline packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: These are unused duplicates of the styles below

Accounts for 5/40 of the new lines

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -170,13 +175,68 @@ exports[`Media Player: Placeholder should render a video placeholder with guidan
   cursor: pointer;
 }
 
+.emotion-1:hover .css-9vh530-StyledPlayButton,
+.emotion-1:focus .css-9vh530-StyledPlayButton {
+  background-color: #B80000;
+}
```
Comment (ryanmccombe) [inline packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: These are unused duplicates of the styles below

Accounts for 5/40 of the new lines

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap [hunk 3]
```diff
@@ -362,6 +372,16 @@ exports[`Media Player: Placeholder should render a video placeholder without dur
...
```
Comment (ryanmccombe) [inline packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: These are unused duplicates of the styles below

Accounts for 5/40 of the new lines

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap [hunk 4]
```diff
@@ -509,6 +524,16 @@ exports[`Media Player: Placeholder should render an audio placeholder 1`] = `
...
```
Comment (ryanmccombe) [inline packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: These are unused duplicates of the styles below

Accounts for 5/40 of the new lines

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap [hunk 5]
```diff
@@ -672,6 +692,16 @@ exports[`Media Player: Placeholder should render an audio placeholder without du
...
```
Comment (ryanmccombe) [inline packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: These are unused duplicates of the styles below

Accounts for 5/40 of the new lines

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap [hunk 6]
```diff
@@ -805,83 +830,44 @@ exports[`Media Player: Placeholder should render an audio placeholder without du
...
 }
 
-@media (min-width:25rem) {
-  .emotion-0 {
-    padding: 1rem;
-  }
+.emotion-1:hover .css-pz3gmj-StyledPlayButton,
+.emotion-1:focus .css-pz3gmj-StyledPlayButton {
+  background-color: #B80000;
 }
```
Comment (ryanmccombe) [inline packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: These are unused quadruplicates of the styles below

Accounts for 15/40 of the new lines

File: packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap [hunk 7]
```diff
@@ -160,67 +150,57 @@ exports[`RadioSchedule should render ltr radio schedules correctly 1`] = `
...
+  .emotion-4 {
     display: block;
+    width: initial;
+    margin: 0;
   }
 
-@media (max-width:14.9375rem) {
-    .emotion-41 {
-      width: initial;
-      margin: 0;
```
Comment (ryanmccombe) [inline packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap]: Emotion seems to be optimising this for us?  These two lines appear 6 times (once in every media query) in the "before"

In the after, they are just specified once, outside any media query

Accounts for 10 of the 40 removed lines

File: packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap [hunk 8]
```diff
@@ -1,112 +1,102 @@
...
     display: grid;
     position: initial;
+    width: initial;
+    margin: 0;
   }
 
-@media (max-width:14.9375rem) {
-    .emotion-161 {
-      width: initial;
-      margin: 0;
```
Comment (ryanmccombe) [inline packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap]: Emotion seems to be optimising this for us? These two lines appear 6 times (once in every media query) in the "before"

In the after, they are just specified once, outside any media query

Accounts for 10 of the 40 removed lines

File: packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap [hunk 9]
```diff
@@ -1233,112 +1213,102 @@ exports[`RadioSchedule should render ltr radio schedules correctly 1`] = `
...
     display: grid;
     position: initial;
+    width: initial;
+    margin: 0;
   }
 
-@media (max-width:14.9375rem) {
-    .emotion-159 {
-      width: initial;
-      margin: 0;
```
Comment (ryanmccombe) [inline packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap]: More optimisation - 12 lines removed, 2 added

File: packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap [hunk 10]
```diff
@@ -1392,67 +1362,57 @@ exports[`RadioSchedule should render rtl radio schedules correctly 1`] = `
...
+  .emotion-4 {
     display: block;
+    width: initial;
+    margin: 0;
   }
 
-@media (max-width:14.9375rem) {
-    .emotion-39 {
-      width: initial;
-      margin: 0;
```
Comment (ryanmccombe) [inline packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap]: Optimised (-12 lines, +2 lines)

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap (hunk 6 above)
Comment (RichardPK) [reply packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap (hunk 5 above)
Comment (RichardPK) [reply packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap (hunk 4 above)
Comment (RichardPK) [reply packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap (hunk 3 above)
Comment (RichardPK) [reply packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (RichardPK) [reply packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌

File: packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (RichardPK) [reply packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap]: Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌

File: .github/workflows/psammead-chromaticqa.yml [hunk 11]
```diff
@@ -17,10 +17,11 @@ jobs:
         node-version: ['12']
     env:
       CI: true
-      CHROMATIC_APP_CODE: ${{ secrets.CHROMATIC_APP_CODE }}
```
Comment (RichardPK) [inline .github/workflows/psammead-chromaticqa.yml:20]: Are these changes expected @ryanmccombe?

Comment (ryanmccombe) [reply .github/workflows/psammead-chromaticqa.yml:20]: aye - Chris added these changes to fix an issue with Chromatic failing on CI

**Review by jroebu14 — APPROVED** [review]

**Review by rebeccamcginn — APPROVED** [review]
Go for it

**Pull request merged**